0-common/schema-ddl/src/main/scala/com.snowplowanalytics/iglu.schemaddl/jsonschema/Linter.scala

```diff
@@ -351,6 +352,28 @@ object Linter {
       }
   }
 
+  final case object duplicateFieldNamesAfterNormalization extends Linter { self =>
+
+    val level: Level = Level.Warning
+
+    case class Details(keys: List[String]) extends Issue {
```

Comment:
Should we highlight the fact that if we have the issue then list of keys is non-empty?

Reply:
In that case using NonEmptyList from Cat would be more appropriate I think ?

Reply:
Exactly!